backend/generator.js

```diff
@@ -25,10 +25,15 @@ exports.executeScript = function (socket, formData, callback) {
   var docPath = formData.docPath === undefined ? '' : formData.docPath;
   var subProject = "[]";
   var subDocpath = [];
```

Comment:
As `subDocpath` is now a string, initialize it with "[]"

Reply:
@pri22296 if it is webui generator. it'll push the doc path and joins as a string `subDocpath = "[" + subDocpath.join(",") + "]";` . please check the else part